test/xmltest/valid.test.js

```diff
@@ -0,0 +1,26 @@
+const xmltest = require('xmltest')
+const {getTestParser} = require('../get-test-parser')
+
+describe('xmltest/valid', () => {
+	describe('standalone', () => {
+		const entries = xmltest.getEntries(
+			xmltest.FILTERS.VALID.SA.files,
+			xmltest.FILTERS.xml
+		)
+		Object.entries(entries).forEach(([pathInZip, filename]) => {
+
+			test(`should match ${filename} with snapshot`, async () => {
+				const input = (await xmltest.getContent(pathInZip))
+					// TODO: The DOCTYPE totally confuses xmldom :sic:
+					// for now we remove it and any newlines after it so we have reasonable tests
+					.replace(/^<!DOCTYPE doc \[[^\]]+]>[\r\n]*/m, '')
+				const expected = await xmltest.getContent(xmltest.RELATED.out(pathInZip))
+				const {errors, parser} = getTestParser()
+
+				const actual = parser.parseFromString(input).toString()
+
+				expect({actual, errors, expected}).toMatchSnapshot()
```

Comment:
I would apply similar changes to what I proposed for `test/xmltest/not-wf.test.js` to give us some cleaner output.

I am not 100% sure whether or not we should show the input in the snapshots. It would be more clutter but could also better show what is going on per case.